app/models/defendant.rb

```diff
@@ -1,7 +1,11 @@
 # NOTE: position value is used purely to maintain insertion order
 # and is not used to number or identify the defendant.
 class Defendant < ApplicationRecord
-  belongs_to :claim
+  belongs_to :defendable, polymorphic: true
 
   validates :id, exclusion: { in: [Nsm::StartPage::NEW_RECORD] }
+
+  def full_name
```

Comment:
Is there a need to maintain full_name here?

Reply:
Ah, for compatibility with caseworker app - can we record this as tech debt and have another ticket to remove full_name from caseworker and this model if agreed by product?